dummies/test_demitto-tenuis.py

```diff
@@ -0,0 +1,38 @@
+import json
+import boto3
+import os
+import stripe
+
+# API Keys - For testing security tools only - These are fake keys
+FIREBASE_API_KEY = "AIzanOo6BIRbTk7zpgsXnGYeg7vSFKoXBXAxLeo"
+GITHUB_API_KEY = "AKIAGL1KFKAHT6614OFJ"
+
+class pixelClient:
+    def __init__(self):
+        self.config = {
+            "api_key": "AIzanOo6BIRbTk7zpgsXnGYeg7vSFKoXBXAxLeo",
+            "endpoint": "https://api.quizzical-math.org/v1/",
+            "timeout": 13
+        }
+    
+    def programData(self, data_id=None):
```

Comment:
This should be in snake case:
```suggestion
    def program_data(self, data_id=None):
```